src/components/share-structure/EditShareStructure.vue

```diff
@@ -257,7 +257,7 @@ export default class EditShareStructure extends Mixins(CurrencyLookupMixin) {
           })
         ) || 'Series name must be unique')
       rules.push(
-        (v: string) => !(v.split(' ').some(r => this.excludedWordsListForSeries.includes(r.toLowerCase()))) ||
+        (v: string) => !(v?.split(' ').some(r => this.excludedWordsListForSeries.includes(r.toLowerCase()))) ||
```

Comment:
Same as above (see line 251).

Is this a problem in the unit tests only, or in deployed code -- how do you reproduce this?

Reply:
This is a problem only in Unit tests I believe. When I ran unit tests without the optional chaining, the two tests I mentioned above were failing.

Reply:
Passing unit tests are awesome, but conditional chaining needs to be used carefully as it can mask actual errors. For example, code could pass when it should actually fail.

See if you can fix the unit test (probably a data initialization problem) instead.